src/ResourceManager/Resources/Commands.Resources/help/Stop-AzureRmDeployment.md

```diff
@@ -0,0 +1,195 @@
+---
+external help file: Microsoft.Azure.Commands.ResourceManager.Cmdlets.dll-Help.xml
+Module Name: AzureRM.Resources
+online version: https://docs.microsoft.com/en-us/powershell/module/azurerm.resources/stop-azurermdeployment
+schema: 2.0.0
+---
+
+# Stop-AzureRmDeployment
+
+## SYNOPSIS
+Cancal a running deployment
+
+## SYNTAX
+
+### StopByDeploymentName (Default)
+```
+Stop-AzureRmDeployment [-Name] <String> [-ApiVersion <String>] [-Pre]
+ [-DefaultProfile <IAzureContextContainer>] [-WhatIf] [-Confirm] [<CommonParameters>]
+```
+
+### StopByDeploymentId
+```
+Stop-AzureRmDeployment -Id <String> [-ApiVersion <String>] [-Pre] [-DefaultProfile <IAzureContextContainer>]
+ [-WhatIf] [-Confirm] [<CommonParameters>]
+```
+
+### StopByInputObject
+```
+Stop-AzureRmDeployment -InputObject <PSDeployment> [-ApiVersion <String>] [-Pre]
+ [-DefaultProfile <IAzureContextContainer>] [-WhatIf] [-Confirm] [<CommonParameters>]
+```
+
+## DESCRIPTION
+The **Stop-AzureRmDeployment** cmdlet cancels a deployment at subscription scope that has started but not completed.
+To stop a deployment, the deployment must have an incomplete provisioning state, such as Provisioning, and not a completed state, such as Provisioned or Failed.
+
+To create a deployment at subscription scope, use the New-AzureRmDeployment cmdlet.
+
+This cmdlet stops only one running deployment. Use the *Name* parameter to stop a specific deployment.
+
+## EXAMPLES
+
+### Example 1
+```
+PS C:\>Stop-AzureRmDeployment -Name "deployment01"
+```
+
+This command cancels a running deployment "deployment01" at the current subscription scope.
+
+### Example 2
+```
+PS C:\>Get-AzureRmDeployment -Name "deployment01" | Stop-AzureRmDeployment
+```
+
```

Comment:
You may also want to add an example on how to get a list of "stoppable" deployments.

Reply:
I think I'll leave it because this is not a common scenario. Usually customer first has a deployment that they want to stop, and they just go ahead and stop it. It's uncommon that they that they only try to stop a deployment after they know it's "stoppable".

Reply:
ok